example/script.js

```diff
@@ -57,4 +57,10 @@ window.onload = async () => {
         copyUInt[i] = i;
     }
     await worker.call("time-transfer", [new Date(), { complexObject: copyUInt }]);
+
+    const canvas = document.getElementById("canvas2D");
+    const offscreenCanvas = canvas.transferControlToOffscreen();
+    if (navigator.userAgent.indexOf("Firefox") != -1) console.warn("This feature (worker context 2d) have a bug in Firefox (see https://bugzilla.mozilla.org/show_bug.cgi?id=801176)");
```

Comment:
!== -1 and better to use bracket for such a long line, but looks good to me

Reply:
Done